panel/models/__init__.py

```diff
@@ -6,7 +6,9 @@
 """
 from .browser import BrowserInfo  # noqa
 from .datetime_picker import DatetimePicker  # noqa
-from .esm import PreactComponent, ReactComponent, ReactiveESM  # noqa
+from .esm import (  # noqa
+    AnyWidgetComponent, PreactComponent, ReactComponent, ReactiveESM,
```

Comment:
But maybe its a better idea to just guide people to convert the widgets to Panel components and share them with the community?

Reply:
The best thing would be if any `anywidget` project would support both a Traitlets/ Jupyter and a Param/ Panel version.

Reply:
Yeah, the autoconversion sounds nice but I'm not sure I want to maintain traitlets -> param mapping and also it falls apart as soon as the class implements methods, e.g.: 

https://github.com/manzt/ipytweakpane/blob/main/src/tweakpane/__init__.py#L14

It would still be possible in that case but you'd have to maintain both the trailets and Param based instances and sync them which seems awkward.